### PR TITLE
#163 added chart for dynatrace-sli-service

### LINKED ISSUE #163: Create a Helm Chart for installing dynatrace-sli-service instead of Kubernetes deployment manifests
In order to align the dynatrace-sli-service with the dynatrace-service, it should provide a Helm Chart for installation (see https://github.com/keptn-contrib/dynatrace-service/issues/261) 

This Helm Chart should have all Environment variables provided as values.

# Definition of Done

- [ ] Make sure that skaffold works with the provided Helm Chart (see solution: https://github.com/keptn/keptn/issues/3421)
- [ ] Helm Chart for dynatrace-sli-service with respective values provided
- [ ] README updated with usage examples
- [ ] When releasing a new version, the Helm Chart needs to be an asset of the release (automation?)
- [ ] Tutorials on tutorials.keptn.sh updated with new install instructions

### PR DESCRIPTION
**Pull request by bacherfl**
Closes #163
Signed-off-by: Florian Bacher <florian.bacher@dynatrace.com>

### CODE REVIEW

File: chart/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+          {{- else }}
+          image: "{{ .Values.dynatraceSliService.image.repository }}:{{ .Values.dynatraceSliService.image.tag | default .Chart.AppVersion }}"
+          {{ end }}
+          imagePullPolicy: {{ .Values.dynatraceSliService.image.pullPolicy }}
+          ports:
+            - containerPort: 80
+          env:
+            - name: CONFIGURATION_SERVICE
+              value: 'http://configuration-service:8080'
+            - name: EVENTBROKER
```
Comment (agrimmer) [inline chart/templates/deployment.yaml]: This ENV variable is outdated. Can we please remove it?

File: deploy/service.yaml [hunk 2]
```diff
@@ -1,106 +0,0 @@
----
```
Comment (agrimmer) [inline deploy/service.yaml]: Please do not delete this file because some automation still uses it.

Comment (bacherfl) [reply deploy/service.yaml]: i have re-added the file

File: chart/templates/deployment.yaml (hunk 1 above)
Comment (bacherfl) [reply chart/templates/deployment.yaml]: sure

**Review by agrimmer — APPROVED** [review]
lgtm

**Pull request merged**